grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -911,14 +911,19 @@ class QueryService implements Status {
                 }
 
                 // Exclude abandoned by default unless statuses are specifically requested
-                if (options.ignoreAbandoned && !options.issueStatusNames) {
+                if (options.ignoreAbandoned && !options.issueStatusNames && !options.freeText) {
                     ne("status", IssueStatus.Abandoned.name)
                 }
 
                 if (options.projectKey) {
                     // For backwards compatibility with existing "ORSP" prefixes, ignore the prefix and like-clause on the identifier
                     like("projectKey", getIssueNumberFromString(options.projectKey))
                 }
+
+                if (options.freeText) {
+                    eq("projectKey", options.freeText)
+                }
+
```

Comment:
This is a pretty big bug. If I search for free text "Church", for example, this will filter out all issues.

Reply:
Yes, I'm aware of that and fixing it.